classes/Task/Update/UpdateFiles.php

```diff
@@ -224,30 +224,32 @@ protected function warmUp(): int
             rename($newReleasePath . DIRECTORY_SEPARATOR . 'install-dev', $newReleasePath . DIRECTORY_SEPARATOR . 'install');
         }
 
-        // Now, we will get the list of changed and removed files between the versions. This was generated previously by
-        // CompareReleases task.
-        $filepath_list_diff = $this->container->getProperty(UpgradeContainer::WORKSPACE_PATH) . DIRECTORY_SEPARATOR . UpgradeFileNames::FILES_DIFF_LIST;
-        $list_files_diff = [];
-
-        // We check if that file exists first and load it
-        if (file_exists($filepath_list_diff)) {
-            $list_files_diff = $this->container->getFileConfigurationStorage()->load(UpgradeFileNames::FILES_DIFF_LIST);
-            // $list_files_diff now contains an array with a list of changed and deleted files.
-            // We only keep list of files to delete. The modified files will be listed in list_files_to_upgrade below.
-            $list_files_diff = $list_files_diff['deleted'];
-
-            // Admin folder name in this deleted files list is standard /admin/.
-            // We will need to change it to our own admin folder name.
-            $admin_dir = trim(str_replace($this->container->getProperty(UpgradeContainer::PS_ROOT_PATH), '', $this->container->getProperty(UpgradeContainer::PS_ADMIN_PATH)), DIRECTORY_SEPARATOR);
-            foreach ($list_files_diff as $k => $path) {
-                if (preg_match('#autoupgrade#', $path)) {
-                    unset($list_files_diff[$k]);
-                } elseif (substr($path, 0, 6) === '/admin') {
-                    // Please make sure that the condition to check if the string starts with /admin stays here, because it was replacing
-                    // admin even in the middle of a path, not deleting some files as a result.
-                    // Also, do not use DIRECTORY_SEPARATOR, keep forward slash, because the path come from the XML standardized.
-                    $list_files_diff[$k] = '/' . $admin_dir . substr($path, 6);
-                }
+        $destinationVersion = $this->container->getState()->getDestinationVersion();
+        $originVersion = $this->container->getState()->getCurrentVersion();
+        $this->logger->debug(sprintf('Generate diff file list between %s and %s.', $originVersion, $destinationVersion));
```

Comment:
Not blocking, but I think we could have used the translator here like the rest of data sent to the logger